peers/open_channels.js

```diff
@@ -186,8 +202,42 @@ module.exports = (args, cbk) => {
         return cbk(null, capacities);
       }],
 
+      // Get the wallet version to make sure the node supports internal funding
```

Comment:
Is there a reason version checking is necessary? I think it can be skipped

Reply:
Just added it because it's unsupported in lower versions. We should just let it fail on its own?

Reply:
Yeah just let it fail, version checking isn't very reliable because people compile themselves

Reply:
Cool ok.